src/robotFactory.js

```diff
@@ -1,10 +1,72 @@
 'use strict';
 
-class BaseRobot {}
+class BaseRobot {
+  constructor(name, weight, coords, chipVersion) {
+    this.name = name;
+    this.weight = weight;
 
-class FlyingRobot {}
+    this.coords = {
+      x: coords.x || 0,
+      y: coords.y || 0,
+    };
 
-class DeliveryDrone {}
+    this.chipVersion = chipVersion;
+  };
+
+  goForward(step = 1) {
+    this.coords.y += step;
+  }
```

Comment:
add empty line between methods. Fix it everywhere)

```suggestion
  }
  
```